docs/interpretation_of_JSON_Schema_draft_7.md

```diff
@@ -18,6 +18,7 @@ The order of interpretation:
 - `items` are interpreted as ether tuples or simple array, where more than 1 item are [merged](#Merging-models). Usage of `items` infers `array` model type.
 - `properties` are interpreted as is, where duplicate `properties` for the model are [merged](#Merging-models). Usage of `properties` infers `object` model type.
 - [allOf](#allOf-sub-schemas)
+- `dependencies` only apply to schema dependencies, since property dependencies adds nothing to the underlying model. Any schema dependencies interpreted and then [merged](#Merging-models) together with the current interpreted model.
```

Comment:
```suggestion
- `dependencies` only apply to schema dependencies, since required dependencies adds nothing to the underlying model. Any schema dependencies are interpreted and then [merged](#Merging-models) together with the current interpreted model.
```

I think that `required dependencies` add logic value to model, but only for languages which support the union types (and in some cases also languages which support intersection types). I remember about your comment https://github.com/asyncapi/modelina/issues/79#issuecomment-857850911 but in the near future we should support it - at least for TS

Reply:
`required dependencies` is not a thing in JSON Schema 🙂 When I refer to property dependencies I refer to this - https://json-schema.org/understanding-json-schema/reference/object.html#property-dependencies

Reply:
Which I assume is what you mean by `required dependencies` 😄

Reply:
Right, sorry 🤦🏼